module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -808,33 +808,35 @@ public function placeHold($holdDetails)
         } catch (Exception $e) {
             throw new ILSException('hold_date_invalid');
         }
-        // Get status to separate Holds (on checked out items) and Pages on a
-        $status = $this->getStatus($holdDetails['item_id']);
         $requestBody = [
-            'requestType' => $item->status->name == 'Available' ? 'Page' : 'Hold',
-            'requestDate' => date('c'),
-            'requesterId' => $holdDetails['patron']['id'],
-            'requester' => [
-                'firstName' => $holdDetails['patron']['firstname'] ?? '',
-                'lastName' => $holdDetails['patron']['lastname'] ?? ''
-            ],
             'itemId' => $holdDetails['item_id'],
+            'requestType' => $holdDetails['status'] == 'Available'
+                ? 'Page' : 'Recall',
```

Comment:
Do we always want to place a "Recall" on an unavailable item, or does this need to be configurable? (I'm assuming that "Hold" is different behavior from "Recall" -- at least, in Voyager, there's the difference between notifying the current holder of the item that they must return it early, vs. simply waiting until they return it and notifying the user with the hold).

Reply:
I wondered about this also.  I'll research and see what I can find.

Reply:
Thanks!

Reply:
This [JIRA ](https://issues.folio.org/browse/UXPROD-1558 ) points to this [Google doc](https://docs.google.com/spreadsheets/d/1jaID-HGft3q4YzJq6Ycy2ejWByAIvou3Zyyw4ko87YI/edit#gid=0) which describes each hold type in detail.

Reply:
Very helpful; that does suggest that there's likely a need for this to be more configurable, since different institutions are going to have different request type preferences. Let me know if you'd like me to help with the design of that.